app/decorators/corporate_category_content_decorator.rb

```diff
@@ -1,11 +1,12 @@
 class CorporateCategoryContentDecorator < CategoryContentDecorator
   def path
+    # TODO: Remove Core namespaced classes when removed
     case object
-    when Core::Article
+    when Core::Article, Mas::Cms::Article
       h.send("#{object.class.to_s.demodulize.underscore}_path", object.id, locale: I18n.locale)
-    when Core::Category
+    when Core::Category, Mas::Cms::Category
       h.send("corporate_#{object.class.to_s.demodulize.underscore}_path", object.id, locale: I18n.locale)
-    when Core::Other, Core::Video
+    when Core::Other, Core::Video, Mas::Cms::Other, Mas::Cms::Video
       "/#{I18n.locale}/#{object.type.pluralize}/#{object.id}"
     end
```

Comment:
Some considerations:

1) IMHO, the best course of action is not adding `TODO` on the code as we have the history on not looking again and leave the code as it is for a long time. Make sure that it is not technical debt and if is inevitable then make sure that there is a card that contains the issue.
2) I am not sure if I missing something but I don't we are using `Mas::Cms::Other` or `Mas::Cms::Video`. Are we using this classes? I would delete this mentions on the code if we are not fetching yet.
3) Since we are already replacing the corporate with the new code why not to delete the Core::Article mention and Core::Category mention?

Reply:
Yeah ok we can probably do this now 👍

Reply:
@tomas-stefano for (3) I looked into this a bit more and I think we should leave `Core::Article` and `Core::Article` there are still quite a few things like ActionPlan that inherit from Article and have not yet been updated and potentially have category stuff in so  I would feel more comfortable addressing this once those controllers have been migrated